kartothek/core/docs.py

```diff
@@ -129,24 +124,45 @@
             When using order sensitive operators on categorical data we will
             assume that the categories obey a lexicographical ordering.
             This filtering may result in less than optimal performance and may
-            be slower than the evaluation on non-categorical data.
-""",
+            be slower than the evaluation on non-categorical data.""",
     "secondary_indices": """
     secondary_indices: List[str]
-        A list of columns for which a secondary index should be calculated.
-""",
+        A list of columns for which a secondary index should be calculated.""",
     "sort_partitions_by": """
     sort_partitions_by: str
-        Provide a column after which the data should be sorted before storage to enable predicate pushdown.
-""",
+        Provide a column after which the data should be sorted before storage to enable predicate pushdown.""",
     "factory": """
     factory: kartothek.core.factory.DatasetFactory
-        A DatasetFactory holding the store and UUID to the source dataset.
-""",
+        A DatasetFactory holding the store and UUID to the source dataset.""",
     "partition_size": """
-    partition_size: int
-        Amount of metapartitions to cluster into one dask partition.
-""",
+    partition_size: Optional[int]
+        Dask bag partition size. Use a larger numbers to decrease scheduler load and overhead, use smaller numbers for a
+        fine-grained scheduling and better resilience against worker errors.""",
+    "metadata_storage_format": """
+    metadata_storage_format: str
+        Optional list of datastorage format to use. Currently supported is `.json` & `.msgpack.zstd"`""",
+    "df_generator": """
+    df_generator: Iterable[Union[pd.DataFrame, Dict[str, pd.DataFrame]]]
+        The dataframe(s) to be stored""",
+    "central_partition_metadata": """
+    central_partition_metadata: TODO
```

Comment:
this keyword should no longer exist (should've been removed in 3.0 already). In any case, it doesn't have any effect anymore. Can you find the usage and add also warnings, please?

Reply:
You are referring to `central_partition_metadata` right? This was used `update_dataset_from_dataframes__iter`. I have depreciated from that place.